pytorch_lightning/core/lightning.py

```diff
@@ -1571,7 +1571,7 @@ def _auto_collect_arguments(cls, frame=None) -> Tuple[Dict, Dict]:
             parents_arguments.update(args)
         return self_arguments, parents_arguments
 
-    def save_hyperparameters(self, *args, frame=None) -> None:
+    def save_hyperparameters(self, *args, ignore: Optional[Union[List, str]] = None, frame=None) -> None:
```

Comment:
```suggestion
    def save_hyperparameters(self, *args, ignore: Optional[Union[List[str], str]] = None, frame=None) -> None:
```

Can you also annotate frame and add the arguments to the docstring?